sdk/keyvault/azure-security-keyvault-certificates/src/private/certificate_constants.hpp

```diff
@@ -17,6 +17,8 @@ namespace Azure { namespace Security { namespace KeyVault { namespace Certificat
     constexpr static const char CertificatesPath[] = "certificates";
     constexpr static const char CertificatesCreatePath[] = "create";
     constexpr static const char IssuersPath[] = "issuers";
+    constexpr static const char ContactsPath[] = "contacts";
+    constexpr static const char ContactsPropertyName[] = "contacts";
```

Comment:
Are these two values intended to be the same string?

Reply:
while they are the same string value , they represent different purposes , one is the path in the url when making a request , the other represents the fiels name in an object.